electron-app/src/components/dashboard/BlockchainBalanceCardList.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+  <v-list-item
+    v-if="amount !== zero"
+    :id="`${name}_box`"
+    :ripple="false"
+    class="blockchain-balance-box__item"
+    @click="doNothing"
+  >
+    <v-list-item-avatar
+      tile
+      class="blockchain-balance-box__icon blockchain-box__icon"
+    >
+      <crypto-icon
+        width="24px"
+        :symbol="blockchainBalanceIcons[name]"
+      ></crypto-icon>
+    </v-list-item-avatar>
+    <v-list-item-content>
+      <v-list-item-title style="display: flex; justify-content: space-between;">
+        <span>
+          {{ name[0].toUpperCase() + name.slice(1) }}
+        </span>
+        <span class="text-end">
+          <amount-display
+            show-currency="symbol"
+            fiat
+            :value="amount"
+          ></amount-display>
+        </span>
+      </v-list-item-title>
+    </v-list-item-content>
+  </v-list-item>
+</template>
+
+<script lang="ts">
+import { default as BigNumber } from 'bignumber.js';
+import { Component, Prop, Vue } from 'vue-property-decorator';
+import CryptoIcon from '@/components/CryptoIcon.vue';
+import AmountDisplay from '@/components/display/AmountDisplay.vue';
+import { Zero } from '@/utils/bignumbers';
+
+@Component({
+  components: { AmountDisplay, CryptoIcon }
+})
+export default class BlockchainBalanceCardList extends Vue {
+  @Prop({ required: true })
+  name!: string;
+  @Prop({ required: true })
+  amount!: BigNumber;
+
+  zero = Zero;
+
+  blockchainBalanceIcons = {
+    bitcoin: 'BTC',
+    ethereum: 'ETC'
```

Comment:
The color when hovering on the ethereum blockchain is the ETC color. We will get stoned to death :smile: 

![2020-05-05-210447_244x248_scrot](https://user-images.githubusercontent.com/1658405/81105438-320c3480-8f14-11ea-87bf-d5e26a5938db.png)

Can you make it just like the color in the normal balances per asset dashboard?

Also I am wondering, how can you explain it visually to the user, without needing any extra explanation, that this is the aggregate of all the value they have in the ethereum blockchain. Not just ether, but all tokens and DeFi too? (I assume it's also DeFi?

Reply:
This is a mistake on my side, it should be `ethereum: 'ETH"` 🤦

Reply:
That's not what I meant by the way. But we can do this in a different PR.

For example if you have DAI locked in the DSR, or ETH locked in maker vaults. We should also add those up here (or not --). If we don't we should have a special entry for assets locked in DeFi.

Reply:
Ah, I get it now. No it does not. Will open a new PR for this.